developers/tutorials/ipfs-filecoin/using-nftstorage.md

```diff
@@ -0,0 +1,451 @@
+---
+description: This tutorial will show how to use NFT.Storage to store NFTs on IPFS and Filecoin for harmony blockchain
+---
+
+# Using NFT.Storage
+
+**NFT.Storage** is a long-term storage service designed for **off-chain** NFT data (like metadata, images, and other assets). Data is [content addressed](https://nftschool.dev/concepts/content-addressing) using **IPFS**, meaning the URL pointing to a piece of data (“ipfs://…”) is completely unique to that data. IPFS URLs can be used in NFTs and metadata to ensure the NFT forever actually refers to the intended data (eliminating things like rug pulls).
```

Comment:
Do we want to include a line like: 
IPFS URLs are storage layer agnostic - meaning you can store your data on [many storage layers at once](https://nft.storage/blog/post/2021-12-14-storage-layer-maximalism/) (whether it be on local storage, cloud storage, or multiple decentralized storage networks.